aya-obj/src/btf/relocation.rs

```diff
@@ -312,13 +312,21 @@ fn relocate_btf_program<'target>(
             // same value, else the relocation is ambiguous and can't be applied
             let conflicts = matches
                 .filter_map(|(cand_name, cand_spec, cand_comp_rel)| {
-                    if cand_spec.bit_offset != target_spec.bit_offset
-                        || cand_comp_rel.target.value != target_comp_rel.target.value
-                    {
-                        Some(cand_name)
-                    } else {
-                        None
+                    if cand_spec.bit_offset != target_spec.bit_offset {
```

Comment:
This seems wrong? Previous condition was `||` now it's `&&`

Reply:
Nice catch oops